apps/protailwind/src/templates/exercise-template.tsx

```diff
@@ -175,10 +180,15 @@ const Video: React.FC<VideoProps> = React.forwardRef(
 )
 
 const GitHubLink: React.FC<{
-  exercise: Exercise
+  exercise: LessonResource
```

Comment:
using LessonResource makes the type more restrictive and forces us to load media at runtime